nautobot/docs/release-notes/version-1.3.md

```diff
@@ -0,0 +1,231 @@
+
+A [data model](../models/circuits/providernetwork.md) has been added to support representing the termination of a circuit to an external provider's network.
+
+#### Python 3.10 Support ([#1255](https://github.com/nautobot/nautobot/pull/1255))
+
+Python 3.10 is officially supported by Nautobot now, and we are building and publishing Docker images with Python 3.10 now.
+
+#### Regular Expression Support in API Filtering ([#1525](https://github.com/nautobot/nautobot/issues/1525))
+
+[New lookup expressions for using regular expressions](../rest-api/filtering.md#string-fields) to filter objects by string (char) fields in the API have been added to all core filters.
+
+The expressions `re` (regex), `nre` (negated regex), `ire` (case-insensitive regex), and `nire` (negated case-insensitive regex) lookup expressions are now dynamically-generated for filter fields inherited by subclasses of `nautobot.utilities.filters.BaseFilterSet`.
+
+#### REST API Token Provisioning ([#1374](https://github.com/nautobot/nautobot/issues/1374))
+
+Introduce the `/api/users/tokens/` REST API endpoint, which includes a child endpoint that can be employed by a user to provision a new REST API token. This allows a user to gain REST API access without needing to first create a token via the web UI.
+
+```bash
+$ curl -X POST \
+-H "Accept: application/json; indent=4" \
+-u "hankhill:I<3C3H8" \
+https://nautobot/api/users/tokens/
+```
+
+This endpoint specifically supports Basic Authentication in addition to the other REST API authentication methods.
+
+#### REST API Versioning ([#1465](https://github.com/nautobot/nautobot/issues/1465))
+
+Nautobot's REST API now supports multiple versions, which may requested by modifying the HTTP Accept header on any requests sent by a REST API client. Details are in the [REST API documentation](../rest-api/overview.md#versioning), but in brief:
+
+- The REST API endpoints that are versioned in the 1.3.0 release are
+    - `/api/extras/jobs/` listing endpoint
+    - `/api/extras/tags/` create/put/patch endpoint
+    - `/api/ipam/ip-addresses/` endpoints
+- All others endpoints are currently un-versioned. However, over time more versioned REST APIs will be developed, so this is important to understand for all REST API consumers.
+- If a REST API client does not request a specific REST API version (in other words, requests `Accept: application/json` rather than `Accept: application/json; version=1.3`) the API behavior will be compatible with Nautobot 1.2, at a minimum for the remainder of the Nautobot 1.x release cycle.
+- The API behavior may change to a newer default version in a Nautobot major release (e.g. 2.0).
+- To request an updated (non-backwards-compatible) API endpoint, an API version must be requested corresponding at a minimum to the Nautobot `major.minor` version where the updated API endpoint was introduced (so to interact with the new Jobs REST API, `Accept: application/json; version=1.3`).
```

Comment:
```suggestion
- The REST API endpoints that are versioned in the 1.3.0 release are
    - `/api/extras/jobs/` listing endpoint
    - `/api/extras/tags/` create/put/patch endpoints
    - all `/api/ipam/ip-addresses/` endpoints
- All other REST API endpoints are currently non-versioned. However, over time more versioned REST APIs will be developed, so this is important to understand for all REST API consumers.
- If a REST API client does not request a specific API version (in other words, requests `Accept: application/json` rather than `Accept: application/json; version=1.3`) the API behavior will be compatible with Nautobot 1.2, at a minimum for the remainder of the Nautobot 1.x release cycle.
- The API behavior may change to a newer default version in a Nautobot major release (such as 2.0).
- To request an updated (non-backwards-compatible) API endpoint, an API version must be requested corresponding at a minimum to the Nautobot `major.minor` version where the updated API endpoint was introduced (so to interact with the updated REST API endpoints mentioned above, `Accept: application/json; version=1.3`).
```